src/ghdcbot/core/social_models.py

```
@@ -0,0 +1,105 @@
+
+class SocialProfileInput(BaseModel):
+    """User input for adding/updating a social profile"""
+    platform: str
+    value: str
+    
+    @field_validator("platform")
+    @classmethod
+    def validate_platform(cls, value: str) -> str:
+        valid_platforms = {"x", "linkedin", "bluesky", "mastodon", "website"}
+        if value.lower() not in valid_platforms:
+            raise ValueError(f"Platform must be one of: {', '.join(valid_platforms)}")
```

Ruff (0.15.18)

[warning] Avoid specifying long messages outside the exception class

(TRY003)